### PR TITLE
Add missing DataGov theme extension 

### PR DESCRIPTION
**Pull request by avdata99**
CKAN is failing at _master_ branch. We need the stable 2.8 branch
Also re-fix nginx config for Airflow (don't know why we lost this changes)

### CODE REVIEW

**Comment by akariv** [comment]
He,  please rebase / sync your branch with the upstream master. There's no need for this PR to contain this many commits.

**Comment by akariv** [comment]
Also, can you point me to the specific failure in master branch - couldn't find it.

File: .docker-compose.datagov-theme.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ services:
     - "5000:5000"
     build:
       args:
+        CKAN_BRANCH: 2.8
```
Comment (akariv) [inline .docker-compose.datagov-theme.yaml]: See my comment below re the failure in master.

File: nginx/default.conf [hunk 2]
```diff
@@ -6,14 +6,15 @@ server {
...
     location /airflow/ {
-        proxy_pass http://harvester:8082;
-
-        proxy_set_header X-Forwarded-For $remote_addr;
         proxy_set_header Host $host;
         proxy_redirect off;
         proxy_http_version 1.1;
         proxy_set_header Upgrade $http_upgrade;
-        proxy_set_header Connection "upgrade";       
+        proxy_set_header Connection "upgrade";
```
Comment (akariv) [inline nginx/default.conf]: Why are these changes needed?
The resolver 127.0.0.11 does not exist on the cluster (and not sure why it's required at all)
the set + proxy_pass is equivalent to the first line you removed... (line 9)

**Review by akariv — CHANGES_REQUESTED** [review]

File: nginx/default.conf (hunk 2 above)
Comment (avdata99) [reply nginx/default.conf]: You asked me to delete the help about this changes: 
https://github.com/avdata99/ckan-cloud-docker/commit/4941f64f89064d599bde8059cf6a798d4bd4aa1c
I did this changes because this code is failing.

File: .docker-compose.datagov-theme.yaml (hunk 1 above)
Comment (avdata99) [reply .docker-compose.datagov-theme.yaml]: It failed locally. _Master_ is having a lot of changes. It's not stable anymore for all extensions.
If you build locally you will see the errors.
The errors in my local machine stops when I add this line and re-build. 
We need to make this work over CKAN 2.8, not on _master_.

**Comment by avdata99** [comment]
> He, please rebase / sync your branch with the upstream master. There's no need for this PR to contain this many commits.

Done @akariv

File: .docker-compose.datagov-theme.yaml (hunk 1 above)
Comment (akariv) [reply .docker-compose.datagov-theme.yaml]: Since the master branch is building in CI, the cause must be some local changes you've made.
If there's some issue with a specific extension you're trying to add, then please open an issue about the errors you're seeing.
Saying "it's not building, I'm getting some errors, install locally and you'll see" is not a good enough explanation.
BTW master branch is using ckan-2.8.1 and not ckan master.

File: nginx/default.conf (hunk 2 above)
Comment (akariv) [reply nginx/default.conf]: Again, this resolver does not exist on the cluster and it's unclear why you would need it just for this route and not for any other route (as this configuration is working as is in the cluster).
If for some reason it's failing this way, please elaborate on the errors you're seeing.

File: .docker-compose.datagov-theme.yaml (hunk 1 above)
Comment (avdata99) [reply .docker-compose.datagov-theme.yaml]: Removed. If I have any error I will define an issue for it.

File: nginx/default.conf (hunk 2 above)
Comment (avdata99) [reply nginx/default.conf]: Back to previous configuration [here](https://github.com/avdata99/ckan-cloud-docker/commit/ed5928cb3a121857c2b5ec729f9b1bcd3a567784).

**Review by akariv — APPROVED** [review]
